backend/removeReported.js

```diff
@@ -0,0 +1,17 @@
+const getReported = require('./__mock__/getReported');
+ 
+function removeReported(toBeRemoved) {
+    var mockReported = [];
+    mockReported = getReported();
+    if (toBeRemoved == null) { //no email specified
+        return mockReported;
+    }
+    var indexToBeRemoved = mockReported.indexOf(toBeRemoved);
+    console.log(indexToBeRemoved);
+    if (indexToBeRemoved > -1) { //if given gmail found within reported list
+        mockReported.splice(indexToBeRemoved, 1); //remove the reported user with the given email from reported list
+    }
+    console.log(mockReported);
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/karatwan/CPEN321_world_resources/pullRequest?prid=9811192)
```suggestion
  console.log(mockReported);
```